frontend/src/components/common/Button/Button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement, MouseEvent, PropsWithChildren } from 'react';
+import styled from 'styled-components';
+
+// @TODO use the barrel index.ts file when bug in https://github.com/typescript-eslint/typescript-eslint/issues/1746 is resolved
+import { orange, grey } from 'styles/colors';
+import { fontSize } from 'styles/typography';
+import { spacing } from 'styles/dimensions';
+
+interface Props {
+  readonly isSecondary?: boolean;
+  onClick?: (event: MouseEvent) => void;
+}
+
+const StyledButton = styled.button<Pick<Props, 'isSecondary'>>`
```

Comment:
```suggestion
const StyledButton = styled.button<{ isSecondary?: boolean }>`
```

Reply:
Out of curiosity, any issues with linking like this? 🤔

Reply:
No, I have not experienced any issues with this. What kind of issues are you thinking about?